lightning/config/config.go

```diff
@@ -290,6 +295,13 @@ func (cfg *Config) Adjust() error {
 		}
 	}
 
+	cfg.TikvImporter.Backend = strings.ToLower(cfg.TikvImporter.Backend)
+	switch cfg.TikvImporter.Backend {
+	case "mysql", "importer":
+	default:
+		return errors.New("invalid config: unsupported `tikv-importer.backend`")
```

Comment:
including the value of `cfg.TikvImporter.Backend` in the error message?

Reply:
seems that those errors from this function not including original value.